working-group-abilities.md

```diff
@@ -0,0 +1,37 @@
+# Working Group Abilities
+
+Established working groups in the OSSF are given broad latitude to accomplish their goals.
+The TAC aims to only require formal approval in limited cases.
+This document outlines some of the explicitly permitted activities and resources made
+available to working groups.
+It is not intended to be exhaustive, and will grow over time.
+
+When in doubt, reach out to the TAC with any questions!
+
+## Repositories
+
+We expect that working groups will need to create repositories for code, tools, and other projects.
+These should be created in the [ossf](github.com/ossf) organization where possible to simplify
+management, but working groups can also create new organizations where necessary.
+
+If any of these projects grow large enough, they can be "spun-off" into separate working groups by
+the TAC.
+As a general rule-of-thumb, if a sub-project requires its own standing meeting, it might be time to
+spin-it-off.
+
+## Papers
+
+As the working groups are intended to be the subject matter experts in their domains,
+they should author papers for release through the OpenSSF with no formal review or approval.
```

Comment:
From the GB level, we will most likely want all external communications (announcements, blogs, papers, etc.) released under the OpenSSF name to go through review and approval by the GB Communications Committee (not yet formed), This is for consistency and alignment with our overall communications strategy.

Reply:
I'm happy to try to clarify the scope here, but I'm -1 on requiring technical whitepapers from WGs to be reviewed by the governing board.

Reply:
I'm fine with a comms committee reviewing. I think what was intended here was that we don't create a gauntlet of redundant reviews where a paper is reviewed in a WG, then the TAC, then another committee....

I think the spirit is that the WG does their own technical/subject matter review, and then the Comms Committee sanity checks no one is doing vendor promotion and guides how the content goes out according to, as Kay says, the comms strategy.

Reply:
Yes - no gauntlets and separating concerns is what I'm trying to do here.

In my head, here's how it would work:

- The WG itself should do final sign off on the whitepaper, and then it's considered "published'. Similar to this one from the [Identifying Threats](https://github.com/ossf/wg-identifying-security-threats/tree/main/publications/threats-risks-mitigations) group.
- The GB (and eventual communications/marketing committee) would then review and promote the work through blog posts, tweets, press releases etc. Realistically I imagine this coordination would be happening somewhat in parallel.

Reply:
The new text below looks like it addresses the comms strategy comment well.
Tweaking this sentence might now be more consistent as 
> ... release through the OpenSSF without review or approval by the TAC.

This wording also removes any unintended interpretation that the papers are not reviewed at all. The working groups should apply their own standards and review processes.

Reply:
Got it! Thanks.